x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/index.tsx

```diff
@@ -266,8 +270,9 @@ export const CreatePackagePolicySinglePage: CreatePackagePolicyParams = ({
   const handleExtensionViewOnChange = useCallback<
     PackagePolicyEditExtensionComponentProps['onChange']
   >(
-    ({ isValid, updatedPolicy }) => {
+    ({ isValid, updatedPolicy, isLoadedPackagePolicies }) => {
       updatePackagePolicy(updatedPolicy);
+      setIsFetchedPackagePolicies(isLoadedPackagePolicies);
```

Comment:
do other extensions also has this problem? maybe we can pass something more generic like `extensionIsLoading`, that why every extension can pass what ever fits their case.